app/packs/controllers/review_workflow_controller.js

```diff
@@ -0,0 +1,17 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  static targets = [ "enabled", "reviewers" ]
+
+  connect() {
+    this.toggle()
+  }
+
+  toggle() {
+    if(this.enabledTarget.checked) {
+      this.reviewersTarget.removeAttribute('disabled')
```

Comment:
This can be reduced to `this.reviewersTarget.disabled = !this.enabledTarget.checked`

Reply:
Huh. I thought the attribute actually had to be removed. But verified that simpler statement works.

Thanks!